### ISSUE TITLE
Support for remote development environments

### ISSUE THREAD

**Issue by tanto259** [IBM/db2forzosdeveloperextension-about#70]
## Description of the feature or enhancement

I would like to see support for remote development environments, such as WSL2 or VSCode Docker container, with the Db2 for z/OS Developer Extension.

This will aid development for people who are using such remote environments in their activities.

At the current state, using a remote development environment with VSCode results in keyring error despite having the pre-requisite installed. Refer to #63 for more information.

## Logs excerpt

At the moment, running VSCode through `code` on the WSL2 terminal will result in this error:
```
[Error - 5:00:14 PM] Db2SqlServiceClient:process:stderr: 2022-02-03T17:00:14,834+0800 [ForkJoinPool.commonPool-worker-3] ERROR: com.github.javakeyring.PasswordAccessException: No stored credentials match db2forzosdeveloperextension/db2conn account: 3445d967-2f72-424e-82fc-dd7732cd3675/user
```

While connecting to WSL2 through a Windows-opened VSCode, will result in this error:
```
[Error - 5:04:19 PM] Db2SqlServiceClient:process:stderr: 2022-02-03T17:04:19,693+0800 [ForkJoinPool.commonPool-worker-3] ERROR: com.github.javakeyring.BackendNotSupportedException: No available keyring backend found
```

**Comment by katsoohoo** [comment]
Hi @tanto259,

With v2.1.0, the extension can now run on remote development environments. Please see our doc [Support for VS Code Remote Development](https://ibm.github.io/db2forzosdeveloperextension-about/docs/tips-and-tricks/support-for-remote-development#support-for-vs-code-remote-development) for more details.

**Comment by katsoohoo** [comment]
I will close this issue since the issues has been resolved in v2.1.0
